docs/docs/mlflow-3/index.mdx

```diff
@@ -279,3 +279,13 @@ MLflow 3.0 introduces some key API changes while also removes some outdated feat
     - mleap
 - AI gateway client APIs: use deployments APIs instead
 
+### Compatibility with MLflow 2.x
+
+We strongly recommend upgrading both client and server to MLflow 3.0.0 for the best experience. However, if you need to use a 2.x client with a 3.x server, or vice versa, here are the compatibility details:
+
+| Client MLflow version | Server MLflow version | Compatibility                        |
+| --------------------- | --------------------- | ------------------------------------ |
+| 2.x                   | 2.x                   | ✅                                   |
+| 3.x                   | 3.x                   | ✅                                   |
+| 2.x                   | 3.x                   | ✅ (not recommended but should work) |
```

Comment:
I feel like we should be `sure` about whether this case really works or not, otherwise we might receive issues/tickets about this. If we don't want to claim or test thoroughly about this case, let's just ask them to keep client & server version consistent with a WARNING or NOTICE instead?

Reply:
Make sense. I'll remove the table and adjust the message.